dev/dll/pch.h

```diff
@@ -32,11 +32,7 @@
 #define MUX_FAIL_FAST() RaiseFailFastException(nullptr, nullptr, 0);
 #define MUX_FAIL_FAST_MSG(MSG) RaiseFailFastException(nullptr, nullptr, 0);
 
-#ifdef BUILD_WINDOWS
-#include <gsl/gsl_util>
-#else
-#include <gsl_util.h>
-#endif
```

Comment:
Can you also remove the GSL nuget package dependency from the MUX project since it's not being maintained?